api/src/openapi/schemas/pagination.ts

```diff
@@ -0,0 +1,79 @@
+import { SchemaObject } from 'ajv';
+
+/**
+ * API schema used to assert pagination query paramaters
+ * for paginated data requests.
+ */
+export const paginationRequestQueryParamSchema: any[] = [
+  {
+    in: 'query',
+    name: 'page',
+    required: false,
+    schema: {
+      type: 'integer',
+      minimum: 1,
+      description: 'The current page number being fetched'
+    }
+  },
+  {
+    in: 'query',
+    name: 'limit',
+    required: false,
+    schema: {
+      type: 'integer',
+      minimum: 1,
+      maximum: 100,
+      description: 'The number of records per page'
+    }
+  },
+  {
+    in: 'query',
+    name: 'sort',
+    required: false,
+    description: 'The column being sorted on'
+  },
+  {
+    in: 'query',
+    name: 'order',
+    required: false,
+    description: 'The order of the sort, i.e. asc or desc'
+  }
```

Comment:
Should this include a schema and define that it must be a string? And ideally also specify an enum for 'asc' and 'desc' (if allowed in the spec).

Reply:
Yeah good call. When I wrote this a while back, I didn't know how to add the schema. It's been added now!